packages/app/src/commands/modernize.js

```diff
@@ -0,0 +1,162 @@
+        describe:
+            'A comma separated list of languages to create files for, when omitted, all are used',
+        type: 'string',
+    },
+
+    deleteOld: {
+        describe:
+            'Deletes the old translation files, use `--no-delete-old` to keep the old files',
+        type: 'boolean',
+        default: 'true',
+    },
+
+    outDir: {
+        describe: 'Output directory for new translation files',
+        type: 'string',
+        default: TRANSLATION_OUT_DIR,
+    },
+
+    inDir: {
+        describe: 'Input directory where old translation files can be found',
+        type: 'string',
+        default: TRANSLATION_IN_DIR,
+    },
+
+    overrideExistingFiles: {
+        describe: 'Overriding the contents of existing translation files',
+        type: 'boolean',
+    },
+
+    appendToExistingFiles: {
+        describe:
+            'Appends the new contents to existing translation files, can only be used when not using `--override-existing-files`',
+        type: 'boolean',
+        conflicts: 'overrideExistingFiles',
+    },
+
+    logMissingKeys: {
+        describe:
+            'Log keys that are present in translation files which are not present in translation file of --primary-language',
+        type: 'boolean',
+        default: 'false',
+    },
+
+    deleteOldFiles: {
+        describe:
+            'Delete the old files that were transformed (will only delete files specified with the `--language` option when present)',
+        type: 'boolean',
+    },
+
+    pootlePath: {
+        describe: 'Set the path for the pootle server',
+        type: 'string',
+    },
+}
+
+const handler = ({
+    inDir,
+    outDir,
+    languages,
+    pootlePath,
+    deleteOldFiles,
+    logMissingKeys,
+    primaryLanguage,
+    overrideExistingFiles,
+    appendToExistingFiles,
+}) => {
+    const languagesToTransform = languages ? languages.split(/,\s*/) : []
+
+    const translationFiles = fs
+        .readdirSync(inDir)
+        .filter(fileIsOldTranslationFile)
+
+    reporter.info('Checking requirements')
+    checkRequirements({
+        inDir,
+        outDir,
+        primaryLanguage,
+        translationFiles,
+    })
+
+    reporter.info('Extracting key/value pairs from translation files')
+    const translations = generateTranslationMappings({
+        inDir,
+        primaryLanguage: primaryLanguage,
+        languagesToTransform,
+        translationFiles,
+    })
+
+    reporter.info('Creating new translation files')
+    createNewTranslationFiles({
+        creationDate: CREATION_DATE,
+        outDir,
+        pootlePath,
+        translations,
+        logMissingKeys,
+        primaryLanguage,
+        languagesToTransform,
+        overrideExistingFiles,
+        appendToExistingFiles,
+    })
+
+    if (deleteOldFiles) {
+        reporter.info('Deleting legacy files')
+        deleteLegacyFiles({
+            translationFiles,
+            languagesToTransform,
+        })
+    }
+}
+
+const modernize = {
```

Comment:
I think the issue here is that you needed a `command` key to tell `yargs` the name of the command.  I restructured it a little bit to match how we do namespacing other places, see #126